examples/tinywl/XdgStackWorkspace.qml

```diff
@@ -81,6 +89,44 @@ Item {
                 }
             }
 
+            OpacityAnimator {
+                id: hideAnimation
+                duration: 300
+                target: surface
+                from: 1
+                to: 0
+
+                onStopped: {
+                    surface.visible = false
+                    if (pendingDestroy)
+                        toplevelComponent.destroyObject(surface)
+                }
+            }
+
+            onMappedChanged: {
+                if (mapped || !waylandSurface.WaylandSocket.rootSocket.enabled) {
```

Comment:
When Socket is enabled and mapped becomes false, set visible after hideAnimation complete， Otherwise set visible directly

Add some comments to make it easier for others to understand

Reply:
No, the show animation can add in future.